src/components/panel/panel.spec.js

```diff
@@ -136,67 +138,69 @@ describe('$mdPanel', function() {
       expect(panelRef.isAttached).toEqual(false);
     });
 
-    it('should resolve when opening/closing', function () {
+    it('should resolve when opening/closing', function() {
       var openResolved = false;
       var closeResolved = false;
 
       expect(panelRef.id).toBeDefined();
       expect(panelRef.open).toBeOfType('function');
       expect(panelRef.close).toBeOfType('function');
 
-      panelRef.open().then(function () {
+      panelRef.open().then(function() {
         openResolved = true;
       });
-      $rootScope.$apply();
+      flushPanel();
 
       expect(openResolved).toBe(true);
       expect(PANEL_WRAPPER_CLASS).toExist();
       expect(panelRef._panelContainer).not.toHaveClass(HIDDEN_CLASS);
       expect(panelRef.isAttached).toEqual(true);
 
-      panelRef.close().then(function () {
+      panelRef.close().then(function() {
         closeResolved = true;
       });
-      $rootScope.$apply();
+      flushPanel();
 
       expect(closeResolved).toBe(true);
       expect(panelRef.isAttached).toEqual(false);
       expect(PANEL_WRAPPER_CLASS).not.toExist();
     });
 
-    it('should reject on create when opening', function () {
+    it('should reject on create when opening', function() {
       var openRejected = false;
 
       panelRef._createPanel = function() {
         return panelRef._$q.reject();
       };
 
-      panelRef.open().then(function() {}, function () {
+      panelRef.open().then(function() {
```

Comment:
Using those `noop` functions is not good here.

You're able to use `panelRef.open().catch(errorCallback)`, which is a shorthand for 
```js
promise.then(null, errorCallback)
```

This can be used in the same lines below.